jwst/assign_wcs/util.py

```diff
@@ -319,9 +320,18 @@ def get_object_info(catalog_name=None):
 
     """
     if catalog_name is None:
-        raise TypeError("Expected name of the catalog file")
+        err_text = "Expected name of the catalog file"
+        log.error(err_text)
+        raise TypeError(err_text)
```

Comment:
I don't believe log.error raises, but logging it helps identify the cause in a place people might look, additionally raising it allows handlers to catch it and halt, specifically here I can't continue without a catalog.....however in looking at this section again, I realize i didn't catch for empty string...updating

Reply:
oh - I didn't really answer....if logging is set to console and emits for error it will go there and the TypeError message will too. I imagine a logging handler could be added to go to file only, then the log file gets updated and the TypeError,unless otherwise directed, goes to the console. I think logging could also be set to only emit for critical, in which case, only the raised error would appear.